test/lib/retire-js/RetireJsEngine.test.ts

```diff
@@ -138,11 +143,31 @@ describe('RetireJsEngine', () => {
 			// We expect the temp directory to still exist, since the process hasn't actually exited yet.
 			const fh = new FileHandler();
 			expect(await fh.exists(tmpDir)).to.equal(true, `Temp directory ${tmpDir} should still exist.`);
+			// Verify that files were actually extracted from AngularJS.zip, and that every JS file has the expected alias.
 			const extractedAngular = await globby(normalize(path.join(tmpDir, '**', 'AngularJS-extracted', '**', '*')));
-			const extractedLeaflet = await globby(normalize(path.join(tmpDir, '**', 'leaflet-extracted', '**', '*')));
-			const extractedRandomMeme = await globby(normalize(path.join(tmpDir, '**', 'RandomMeme-extracted', '**', '*')));
 			expect(extractedAngular.length).to.be.greaterThan(0, 'Should be some copied Angular files');
+			for (const subpath of extractedAngular) {
+				if (path.extname(subpath).toLowerCase() === '.js') {
```

Comment:
It would be good to add a counter or boolean to ensure this if statement is entered by the test. This avoids the test passing because the if body never executed. Same applies below.

Reply:
Good catch.